### PR TITLE
utils_disk: New API to get windows drive letters of specified disk

### PR DESCRIPTION
**Pull request by YongxueHong**
New API to get the drive letters of the specified disk in windows
guest.

ID: 1876750
Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

File: virttest/utils_disk.py [hunk 1]
```diff
@@ -790,6 +790,36 @@ def get_disk_size(session, os_type, did):
         return get_disk_size_windows(session, did)
 
 
+def get_drive_letters(session, did):
+    """
+    Get drive letters of specified disk in windows guest.
+
+    :param session: Windows VM session.
+    :type session: aexpect.ShellSession
+    :param did: The disk index.
```
Comment (qingwangrh) [inline virttest/utils_disk.py:799]: It looks like the paramater is the physical disk index? how to handle if it has multi partitions

Comment (YongxueHong) [reply virttest/utils_disk.py:799]: I will get all the letters.

Comment (YongxueHong) [reply virttest/utils_disk.py:799]: Hi @qingwangrh 
BTW, the name of the function is `get_drive_letters`, so it means get all the drive letters.
Thanks.

**Comment by qingwangrh** [comment]
python3 ConfigTest.py --testcase=virtio_storage_in_use.with_block.data_disk.with_system_reset.after_bg_test --guestname=Win2019 --platform=x86_64 --driveformat=virtio_blk,virtio_scsi --machines=q35 --clone=no

(1/1) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_storage_in_use.with_block.data_disk.with_system_reset.after_bg_test.q35: PASS (464.78 s)

LGTM

**Comment by YongxueHong** [comment]
Hi @luckyh 
Please help me to review, thanks.

**Review by luckyh — APPROVED** [review]
ACK

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**